pp3/module/Application/src/Application/Entity/Verification.php

```diff
@@ -65,11 +65,15 @@ public function getStatusBadgeClass() {
 
     public function getStatusBadgeTitle() {
         if ($this->status === self::STATUS_NOGO) {
-            return 'NoGo';
+            return 'Verification declined';
```

Comment:
I would prefer **Verification failed** wording but **declined** is probably also clear.